packages/web/README.md

```diff
@@ -33,8 +33,8 @@ in your HTML template:
 
 ### Advanced Implementation in Product with Sass
 
-❗ **Important:** Make sure you have
-[configured Sass load path][configuring-load-path] for `@tokens` and
+❗ **Important:** Make sure you have `sass` dependency installed in your project (`sass` is mark as an optional peer dependency since you can use the styles directly from CDN).
```

Comment:
Yes, and we even distribute the CSS:

```suggestion
❗ **Important:** Make sure you have `sass` dependency installed in your project (`sass` is marked as optional peer dependency since you can use the pre-built distribution CSS).
```